katalog/istio-operator/operator/README.md

```diff
@@ -0,0 +1,14 @@
+## Istio operator
+
+Requirements:
+
+- [kubernetes-split-yaml](https://github.com/mogensen/kubernetes-split-yaml)
+
+The current manifests were generated using:
```

Comment:
I agree with the approach on documenting how the manifests were generated, i would put these infos on a different file, maybe `MAINTENANCE.md`